pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -283,12 +283,12 @@ func (os *OpenStack) getVolume(diskName string) (Volume, error) {
 }
 
 // Create a volume of given size (in GiB)
-func (os *OpenStack) CreateVolume(name string, size int, vtype, availability string, tags *map[string]string) (volumeName string, err error) {
+func (os *OpenStack) CreateVolume(name string, size int, vtype, availability string, tags *map[string]string) (volumeName string, zone string, err error) {
```

Comment:
You've called the input `availability` and the output `zone`, even though they have the same value (when non-empty).  Can we pick one term or the other?